### PR TITLE
remove plugin-specific import CTF protocol

### PR DESCRIPTION
**Pull request by azazellochg**
Goes together with https://github.com/scipion-em/scipion-em-tomo/pull/439
@fede-pe  please let me know if you are happy with the suggested changes so I can proceed to remove addCTFTomoSeriesToSetFromDefocusFile function and adapt existing estimation protocols code.

### CODE REVIEW

**Comment by fede-pe** [comment]
Hi @azazellochg, I like the idea. I added @pconesa and @JorMaister so they can check the implementation too.

Besides that sonarCloud is not passing due to code duplication but I have no idea how to fix it.

**Comment by azazellochg** [comment]
Thanks! I'll fix the duplication.

**Comment by azazellochg** [comment]
All fixed now and ready for review

File: imod/tests/test_protocols_imod.py
Comment (JorMaister) [inline imod/tests/test_protocols_imod.py:1]: TestImodCTFCorrectionWorkflow is not passing, at least on my machine. 

I got the following error:

[   FAILED ] TestImodCTFCorrectionWorkflow.setUpClass


Traceback (most recent call last):
  File "/home/jjimenez/anaconda3/envs/scipion3/lib/python3.8/unittest/suite.py", line 163, in _handleClassSetUp
    setUpClass()
  File "/home/jjimenez/scipion3/scipion-em-imod/imod/tests/test_protocols_imod.py", line 698, in setUpClass
    cls._runImportSetOfCtfSeries(filesPath=os.path.split(cls.inputCtfFile)[0],
  File "/home/jjimenez/scipion3/scipion-em-imod/imod/tests/test_protocols_imod.py", line 241, in _runImportSetOfCtfSeries
    cls.protImportSetOfCtfSeries = cls.newProtocol(tomo.protocols.ProtImportTsCTF,
AttributeError: module 'tomo.protocols' has no attribute 'ProtImportTsCTF'

File: imod/__init__.py
Comment (JorMaister) [inline imod/__init__.py:1]: Just to keep you updated regarding the versioning, we've started to use this semantic versioning recently:

https://semver.org/

Following this, I'd say it's a Minor or even a Major change.

**Review by JorMaister — CHANGES_REQUESTED** [review]

File: imod/tests/test_protocols_imod.py
Comment (azazellochg) [reply imod/tests/test_protocols_imod.py:1]: You need to use this with https://github.com/scipion-em/scipion-em-tomo/pull/439

File: imod/__init__.py
Comment (azazellochg) [reply imod/__init__.py:1]: Good point, I bumped it to 3.4

**Review by JorMaister — APPROVED** [review]

**Review by pconesa — APPROVED** [review]

**Pull request merged**